projects/VSCode/.vscode/launch.json

```diff
@@ -8,7 +8,7 @@
             "name": "Debug",
             "type": "cppdbg",
             "request": "launch",
-            "program": "${workspaceFolder}/main",
+            "program": "${workspaceFolder}/game",
```

Comment:
this change shouldn't be here, I corrected it some commits ago...

Reply:
Why is that though? When i debug it produces an executable named 'game'. Why do you think it should be 'main'? If you look at Windows it also says `"${workspaceFolder}/game.exe"`

Reply:
@stefanerwinmayer On Windows it generates `main.exe`, checking `tasks.json`, it's configured this way on `build release windows`. 

I think that's the right way because compiled file name can change.

Reply:
Ok, checking a bit further it seems `main.c` is hardcoded in `Makefile`...

Reply:
Ok I can revert this one. Do you want the other change or should we close this and I apply the make fix change to any makefile that needs it and open a new PR?

Reply:
Just revert this change and I'll merge the other one. Yes, it should be applied to all Makefiles, it can be done in another PR.